### PR TITLE
added error log to be printed when command exit code is non zero

### PR DESCRIPTION
**Pull request by adarsh-jaiswal**
Small change: added command execution error log messages.

### CODE REVIEW

**Review by chriswr95 — CHANGES_REQUESTED** [review]

File: src/main/java/graphql/testrunner/service/CommandExecutorService.java [hunk 1]
```diff
@@ -89,8 +94,8 @@ ProcessBuilder getBuilder(String dir) {
         }
     }
 
-    BufferedReader getReader(Process p) {
-         return new BufferedReader(new InputStreamReader(p.getInputStream()));
+    BufferedReader getReader(Process p, Function<Process, InputStream> func) {
+         return new BufferedReader(new InputStreamReader(func.apply(p)));
```
[the remark this answers is not in the document]
Comment (adarsh-jaiswal) [reply src/main/java/graphql/testrunner/service/CommandExecutorService.java:105]: done

File: src/main/java/graphql/testrunner/service/CommandExecutorService.java [hunk 2]
```diff
@@ -61,13 +63,16 @@ private String readOutput(Process process) throws TestRunnerException {
...
             if (exitVal == 0) {
-                LOGGER.log(Level.INFO, "Command logs : {0}", output);
+                String successLog = readOutput(process, Process::getInputStream);
+                LOGGER.log(Level.INFO, "Command logs : {0}", successLog);
             } else {
-                LOGGER.log(Level.SEVERE, "Error exit code on command :{0}", command);
+                String errorLog = readOutput(process, Process::getErrorStream);
+                LOGGER.log(Level.SEVERE, "Error exit code on command : {0}", command);
+                LOGGER.log(Level.SEVERE, "Error message: {0}", errorLog);
                 throw new TestRunnerException();
```
[the remark this answers is not in the document]
Comment (adarsh-jaiswal) [reply src/main/java/graphql/testrunner/service/CommandExecutorService.java]: added.

File: src/main/java/graphql/testrunner/service/CommandExecutorService.java [hunk 3]
```diff
@@ -73,7 +73,7 @@ private void writeOutput(Process process, List<String> command) throws IOExcepti
                 String errorLog = readOutput(process, Process::getErrorStream);
                 LOGGER.log(Level.SEVERE, "Error exit code on command : {0}", command);
                 LOGGER.log(Level.SEVERE, "Error message: {0}", errorLog);
-                throw new TestRunnerException();
+                throw new TestRunnerException("Command failed to execute.");
```
Comment (chriswr95) [inline src/main/java/graphql/testrunner/service/CommandExecutorService.java:76]: please pass the error message in here too

Comment (adarsh-jaiswal) [reply src/main/java/graphql/testrunner/service/CommandExecutorService.java]: done

**Review by chriswr95 — APPROVED** [review]

**Pull request merged**